pkgs/servers/tacacs+/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, tcp_wrappers, flex, bison, perl }:
+
+stdenv.mkDerivation rec {
+  pname = "tacacs+";
+  version = "4.0.4.28";
+
+  src = fetchurl {
+    url = "ftp://ftp.shrubbery.net/pub/tac_plus/tacacs-F${version}.tar.gz";
+    sha256 = "17i18z3s58c8yy8jxp01q3hzz5nirs4cjxms18zzkli6ip4jszql";
+  };
+
+  buildInputs = [ tcp_wrappers flex bison perl ];
+
+  meta = with stdenv.lib; {
+    description = "TACACS+ authentication daemon";
+    homepage = "http://www.shrubbery.net/tac_plus/";
+    license = [ "cisco" ];
```

Comment:
If this license isn't included in `licenses.` maybe you should add it.

Reply:
Maybe I should base this PR on https://github.com/facebook/tac_plus. There it is licensed under bsd3.

Reply:
Using license.free for this package.